frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -433,6 +439,11 @@ export const keyMapping: KeyMappingInterface = {
     },
 }
 
+export const ENVIRONMENTS = {
```

Comment:
Think this might be better as an `enum`? I would also keep this in `constants.tsx` as it's shared across files